app/Livewire/Server/Show.php

```
@@ -361,6 +374,87 @@ public function instantSave()
+    }
+
+    public function startHetznerServer()
+    {
+        try {
+            if (! $this->server->hetzner_server_id || ! $this->server->cloudProviderToken) {
+                $this->dispatch('error', 'This server is not associated with a Hetzner Cloud server or token.');
+
+                return;
+            }
+
+            $hetznerService = new \App\Services\HetznerService($this->server->cloudProviderToken->token);
```

PHPMD (2.15.0)

Missing class import via use statement (line '446', column '35'). (undefined)

(MissingImport)